schema/verify_schema.py

```diff
@@ -91,6 +91,15 @@ def testJsonSchemaObject(schema_file, test_file, schemaObject):
 errors += testJsonSchema('PalRecord.schema.json', 'PalRecordExample.json')
 
 errors += testJsonSchema('FullActivityDump.schema.json', 'FullActivityDumpExample.json')
+
+errors += testJsonSchema('InstallationParamData.schema.json', 'InstallationParamDataExample.json')
+
+errors += testJsonSchema('CbsdRecordData.schema.json', 'CbsdRecordDataExample.json')
+
+errors += testJsonSchema('GrantRecord.schema.json', 'GrantRecordExample.json')
+
+errors += testJsonSchema('zoneData.schema.json', 'zoneDataOfPpaExample.json')
```

Comment:
Minor: Should this file name start with capital Z to match the other naming conventions?

Reply:
thank you for your comment, I tried to fix this, but it seems git on windows doesnt see this difference of the file name.
Could you check it ?